src/app.js

```diff
@@ -85,6 +86,17 @@ var app = function(options) {
     app.use(sslify.HTTPS(options.trustProxy));
   }
 
+  // When we force SSL, we also want to set the HSTS header file correctly.  We
+  // also want to allow testing code to check for the HSTS header being
+  // generated correctly without having to generate an SSL cert and key and
+  // have express listen on ssl
+  if (options.forceSSL || options.forceHSTS) {
```

Comment:
Can we remove the `options.forceHSTS` and just default to having it on if `forceSSL`...

You can still test it using `forceSSL` and `trustProxy`.